examples/6tisch/etsi-plugtest-2017/sf-plugtest.c

```diff
@@ -402,9 +409,10 @@ delete_req_handler(const linkaddr_t *peer_addr,
   assert(cell_list_len == sizeof(sf_plugtest_cell_t));
   memcpy(&pending_cell, cell, sizeof(pending_cell));
   timeslot = pending_cell.slot_offset[0] + (pending_cell.slot_offset[1] << 8);
+  channel_offset = pending_cell.channel_offset[0] + (pending_cell.channel_offset[1] << 8);
 
   if((slotframe = tsch_schedule_get_slotframe_by_handle(0)) == NULL ||
-     (link = tsch_schedule_get_link_by_timeslot(slotframe, timeslot)) == NULL ||
+    (link = tsch_schedule_get_link_by_timeslot(slotframe, timeslot, channel_offset)) == NULL ||
```

Comment:
Don't we need a heading white space here for the correct indentation? Similar things are seen in other places.